MegaQuests/src/main/java/com/elmer/megaquests/commands/QuestGUICommand.java

```diff
@@ -115,24 +99,32 @@ public void createQuestGUI(Player player){
                 taskAmounts.add(taskAmount);
             }
         } else {
-            System.out.println("MegaQuests: Too many quests are disabled, Enable more in quests.yml!");
+            Bukkit.getLogger().log(Level.WARNING, "MegaQuests: Too many quests are disabled, Enable more in quests.yml!");
         }
 
         createQuestTimer();
 
         player.openInventory(questGUI);
 
 
-
     }
 
     public void openQuestGUI(Player player) {
 
-        int slot = megaQuests.getQuestManager().getStartingSlot();
-        for (int i = 0; i < questItems.size(); i++){
-            questItems.set(i,new ItemBuilder(questsData.get(i).getItemDisplay())
-                    .withDisplayName(questsData.get(i).getDisplay() +  " " + ChatColor.GRAY + megaQuests.getQuestManager().getProgress(player.getUniqueId(), questsData.get(i))  + "/" + taskAmounts.get(i))
-                    .withLore(ChatColor.GRAY + "Completion Reward $" + questsData.get(i).getReward()).build());
+        int slot = questManager.getStartingSlot();
+        for (int i = 0; i < questItems.size(); i++) {
+            String itemDisplayName = questsData.get(i).getDisplay() +
+                    " " +
+                    ChatColor.GRAY +
+                    questManager.getProgress(player.getUniqueId(), questsData.get(i)) +
+                    "/" +
+                    taskAmounts.get(i);
+
+            ItemStack questItem = new ItemBuilder(questsData.get(i).getItemDisplay())
+                    .withDisplayName(itemDisplayName)
+                    .withLore(ChatColor.GRAY + "Completion Reward $" + questsData.get(i).getReward()).build();
+
+            questItems.set(i, questItem);

```

Comment:
You have the creation of `ItemStack` inside the second argument method. 
Take the initialization of this before passing it as a parameter, as it's hard to read.
`StringBuilder` is better than concatenation. Find information about it on the internet.